Public/Get/Get-HaloSoftwareLicence.ps1

```diff
@@ -58,27 +61,43 @@ function Get-HaloSoftwareLicence {
             $QSCollection = New-HaloQuery -CommandName $CommandName -Parameters $Parameters
             $Resource = "api/SoftwareLicence/$($LicenceID)"
             $RequestParams = @{
-                Method = 'GET'
-                Resource = $Resource
+                Method          = 'GET'
+                Resource        = $Resource
                 AutoPaginateOff = $True
-                QSCollection = $QSCollection
-                ResourceType = 'licences'
+                QSCollection    = $QSCollection
+                ResourceType    = 'licences'
             }
-        } else {
+        }
+        elseif ($IncludeInactive) {
+            Write-Verbose 'Running in multi-item mode with inactive items included.'
+            $QSCollection = New-HaloQuery -CommandName $CommandName -Parameters $Parameters -IsMulti
+            $Resource = 'api/SoftwareLicence'
+            $RequestParams = @{
+                Method          = 'GET'
+                Resource        = $Resource
+                AutoPaginateOff = $Paginate
+                QSCollection    = $QSCollection
+                ResourceType    = 'licences'
+                includeinactive = $True
+            }
+            
+        }
+        else {
```

Comment:
```suggestion
        } else {
```
Parameters are automatically and dynamically added to the query string using the custom Query String processor function. No need to create separate parameter blocks for each.